### PR TITLE
Fix docs bugs

### PR DESCRIPTION
**Pull request by ChengYueJia**
Fix docs bugs to make them readable
-  fix the files names with the wrong md-suffix, which ends with .mdx by renaming the .mdx-file to .md-file
- Fix the wrong link in readme

## Description
Fix docs bugs to make them readable
-  fix the files names with the wrong md-suffix, which ends with .mdx by renaming the .mdx-file to .md-file
- Fix the wrong link in readme

Fixes pyrsia/pyrsia#

This PR does... by accomplishing... and it can be reviewed by... you can also test the changes by running...

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [x] I've built the code `cargo build --all-targets` successfully.
- [x] I've run the unit tests `cargo test --workspace` and everything passes.
- [x] I've made sure my rust toolchain is current `rustup update`.

### CODE REVIEW

**Comment by betarelease** [comment]
Hi @ChengYueJia 

Thanks for sending this PR. We appreciate your help.
I will comment on the specifics of this PR in the review.

File: docs/social.md [hunk 1]
```diff
@@ -32,12 +32,12 @@ Introducing a new OSS decentralized container registry solution called Pyrsia. P
 ## Calendar
 
 export const GoogleCalendar = () => {
-    return (
```
Comment (betarelease) [inline docs/social.md:35]: I see that you modified the extension from `.mdx` to `.md`.

We use docusaurus and its parser to show the documentation on the website. docusaurus use mdx.

Here is more info https://docusaurus.io/docs/markdown-features
https://mdxjs.com/

Please update the PR to revert this rename.

File: docs/readme.md [hunk 2]
```diff
@@ -10,5 +10,5 @@ and get involved!
 
 <!-- markdown-link-check-disable -->
 
-- [Tutorials](/docs/tutorials/)
```
Comment (betarelease) [inline docs/readme.md:13]: Thanks for catching these issues.

**Review by betarelease — COMMENTED** [review]
I requested some changes. Please update and I will review and merge.

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

File: docs/social.md (hunk 1 above)
Comment (ChengYueJia) [reply docs/social.md]: Thx for pointing it out.
I've fixed it be merge my commit.
So you can see this in the new pr #1024

**Pull request closed without merging**